### DISCUSSION TITLE
how to change cmdline cursor width and color?

### DISCUSSION THREAD

**Discussion by marcoSven** [folke/noice.nvim#510] [Q&A]
My guicursor config works in the classic cmdline but it is ignored in the cmdline_popup

`vim.opt.guicursor =
    "n:block-Cursor/lCursor,v:block-VCursor/lVCursor,c:ver25-CCursor/lCCursor,i-ci-ve:ver25-ICursor/lICursor,r-cr:hor20,o:hor50"`

**Comment by chrisgrieser** [comment]
Apparently an upstream limitation and nothing we can do 😕 
https://github.com/folke/noice.nvim/issues/552
